.github/workflows/staker-rewards.yml

```diff
@@ -0,0 +1,80 @@
+name: Staker Rewards
+
+#TODO: extend the conditions once workflow gets tested together with other workflows
+on:
+  push:
+    branches:
+      # TODO: Run on master after we're fully migrated from Circle CI
```

Comment:
We can switch completely to GH Actions for staker-rewards checks as it's independent and no other CircleCI jobs rely on them.

Reply:
So let's remove all the CircleCI config for staker-rewards.

Reply:
Right, done.